# Towards the identification of Alpha sy-nuclein Drosophila model of Parkinson’s disease by Combining Mass-spectrometry based proteomics and Artificial Intelligence
Till date, two key characteristics of Parkinson’s disease (PD) have been consistently re-
ported by scientists: the presence of alpha-synuclein (a-syn) positive Lewy Bodies (LD)
in the brainstem, alongside the degeneration of dopaminergic neurons in the substantia
nigra pars compacta. Some researchers had postulated that PD might result from gene
dosage effects of SNCA, and p.A53T is primarily related to the rapid progression of rel-
atively early onset of PD. Mass-spectrometry-based proteomics approaches have proven
essential in quantifying proteins and peptides in a bias-free fashion, aiding in investigat-
ing gene expression in specific conditions throughout the life cycle of a cell or group of
cells. Here, we develop a methodology combining the analysis of the Drosophila model
of Parkinson’s disease samples by mass-spectrometry with machine learning. The exper-
iment was carried out using several classic but successful ML models, including Support
Vector Machine (SVM), Random Forest, Gradient Boost Classifier, Logistic Regression,
K Nearest Neighbor, and Decision Tree models. Although considering a single feature
such as peptide intensity for the machine learning classifier models provided 65-69% ac-
curacy of detecting PD samples, it increased by about 15-20% when we integrated the
peptide count as an additional feature along with intensity of the peptides. Notably, three
of our prominent models, namely KNN, Decision Tree and Logistic Regression gives us
the accuracy more than 72% and this paramount level of accuracy ensures that our mod-
els can approximately classify whether samples might have the disease or not. These
findings imply that using Machine Learning methods to analyze MS based PD samples
is a straightforward, safe, rapid, and cost-effective way to detect Parkinson’s disease.

